batch/plugins/org.jboss.tools.batch.ui/src/org/jboss/tools/batch/ui/editor/internal/model/JobXMLEditor.sdef

```diff
@@ -200,21 +211,7 @@
 			<section>
 				<label>Processor</label>
 				<content>
-					<property-editor>
- 		               <hint>
- 		                   <name>assist.contributors</name>
-		                   <value>OpenRefAssistContributor</value>
-                		</hint>
-                        <action>
-                           <id>Sapphire.Jump</id>
-                           <label>Open</label>
-                        </action>
-                        <action-handler>
-                            <action>Sapphire.Jump</action>
-                            <impl>org.jboss.tools.batch.ui.editor.internal.action.OpenOrCreateArtifactActionHandler</impl>
```

Comment:
Yes, it is. It is impossible to explain looking at diff, because in some cases like this it completely confuses. What I did is that I declared a separate definition 
<property-editor>
   <id>RefPropertyEditor</id>
   ---
   <property>Ref</property>
</property-editor>
which includes previous features (content assist and jump action) and new feature (browse action), and then used it in 16 places as 
<include>RefPropertyEditor</include>
That reduced file by about 200 lines instead of increasing it by about 150 lines.